packages/react-notifications/src/components/notifications-list/notifications-list.tsx

```diff
@@ -0,0 +1,273 @@
+                  reportErrorToSentry(
+                    new Error(
+                      `@commercetools-frontend/react-notifications. Invalid prop "kind" supplied: ${notification.kind}.`
+                    ),
+                    { extra: notification }
+                  );
+                }
+                return null;
+              }
+            }
+          })}
+        </div>
+      );
+    }
+    case NOTIFICATION_DOMAINS.PAGE: {
+      const notifications = useSelector<TAppState, TAppNotificationPage[]>(
+        selectPageNotifications
+      );
+      return (
+        <div css={getStyles(props)}>
+          {notifications.map(notification => {
+            // 1. Check if there is a custom notification component first
+            const CustomNotificationComponent = mapCustomNotificationToComponent(
+              notification
+            ) as Notification;
+            if (CustomNotificationComponent) {
+              return (
+                <CustomNotificationComponent
+                  key={notification.id}
+                  notification={notification}
+                  dismiss={() => {
+                    dispatch(removeNotification(notification.id));
+                  }}
+                />
+              );
+            }
+
+            switch (notification.kind) {
+              case NOTIFICATION_KINDS_PAGE.error:
+              case NOTIFICATION_KINDS_PAGE.warning:
+              case NOTIFICATION_KINDS_PAGE.info:
+              case NOTIFICATION_KINDS_PAGE.success: {
+                const { values, ...genericNotification } = notification;
+                return (
+                  <GenericNotification
+                    key={notification.id}
+                    notification={{
+                      ...genericNotification,
+                      kind: notification.kind,
+                    }}
+                    dismiss={() => {
+                      dispatch(removeNotification(notification.id));
+                    }}
+                  />
+                );
+              }
+              case NOTIFICATION_KINDS_PAGE['api-error']: {
+                const { text, ...errorNotification } = notification;
+                return (
+                  <ApiErrorNotification
+                    key={notification.id}
+                    notification={errorNotification}
+                    dismiss={() => {
+                      dispatch(removeNotification(notification.id));
+                    }}
+                  />
+                );
+              }
+              case NOTIFICATION_KINDS_PAGE['unexpected-error']: {
+                const { text, ...errorNotification } = notification;
+                return (
+                  <UnexpectedErrorNotification
+                    key={notification.id}
+                    notification={{
+                      ...errorNotification,
+                      values: notification.values as TAppNotificationValuesUnexpectedError,
```

Comment:
This is also weird, and I don't fully understand why, but TS things that the `values` that we pass is of type `TAppNotificationValuesApiError`, which is wrong.